apps/content/api/public/recitations.py

```
@@ -41,15 +68,13 @@ class RecitationFilter(FilterSchema):
 @ordering(ordering_fields=["name", "created_at", "updated_at"])
 @searching(search_fields=["name", "description", "resource__publisher__name", "reciter__name"])
 def list_recitations(request, filters: RecitationFilter = Query()):
```

Ruff (0.14.10)

Unused function argument: `request`

(ARG001)